luigi/contrib/prometheus_metric.py

```diff
@@ -1,63 +1,82 @@
 from prometheus_client import CollectorRegistry, Counter, Gauge, generate_latest, CONTENT_TYPE_LATEST
+from luigi import parameter
 from luigi.metrics import MetricsCollector
+from luigi.task import Config
+
+
+class prometheus(Config):
+    use_task_family = parameter.BoolParameter(default=True, parsing=parameter.BoolParameter.EXPLICIT_PARSING)
+    task_parameters = parameter.ListParameter(default=[])
 
 
 class PrometheusMetricsCollector(MetricsCollector):
 
+    def _generate_task_labels(self, task):
+        return {
+            label: task.family if label == "family" else task.params.get(label)
+            for label in self.labels
+        }
+
     def __init__(self):
         super(PrometheusMetricsCollector, self).__init__()
         self.registry = CollectorRegistry()
+        config = prometheus()
```

Comment:
I'm not sure why this is the case (probably a miss during PR review), but the initial implementation of this `PrometheusMetricsCollector` forgot to allow for `*args, **kwargs` to its `__init__()`.

In most other classes in Luigi, we allow for args/kwargs and pass `**kwargs` to the config class initializations to facilitate config overrides. e.g. https://github.com/spotify/luigi/blob/master/luigi/contrib/datadog_metric.py#L27

Reply:
I added init params and it was super useful in unit tests!